frontend_app_file/src/container/File.jsx

```diff
@@ -184,23 +183,17 @@ export class File extends React.Component {
     }
   }
 
-  handleContentCommentCreated = (data) => {
-    if (data.fields.content.parent_id === this.state.content.content_id) {
-      const sortedNewTimeLine = sortTimelineByDate([
-        ...this.state.timeline,
-        {
-          ...data.fields.content,
-          created_raw: data.fields.content.created,
-          created: displayDistanceDate(data.fields.content.created, this.state.loggedUser.lang),
-          timelineType: data.fields.content.content_type,
-          hasBeenRead: data.fields.client_token === this.sessionClientToken
-        }
-      ])
-      this.setState({
-        timeline: sortedNewTimeLine,
-        isLastTimelineItemCurrentToken: data.fields.client_token === this.sessionClientToken
-      })
-    }
+  handleContentCommentCreated = (tlm) => {
+    const { props, state } = this
+    // Not a comment for our content
+    if (tlm.fields.content.parent_id !== state.content.content_id) return
```

Comment:
We've had bugs because we weren't casting these properties in the past. Are we sure those are both numbers or both strings, and that the case where one is 0 or empty is correctly handled?

(see `frontend_app_html-document/src/container/PopupCreateHtmlDocument.jsx` line 49 for instance:  `if (Number(data.content.parent_id) !== Number(state.folderId) …`)